<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Data driven Framework" duration-ms="13182" started-at="2019-06-12T13:04:44Z" finished-at="2019-06-12T13:04:57Z">
    <groups>
    </groups>
    <test name="Real Testing" duration-ms="13182" started-at="2019-06-12T13:04:44Z" finished-at="2019-06-12T13:04:57Z">
      <class name="TestScript.TC1">
        <test-method status="PASS" signature="setup()[pri:0, instance:TestScript.TC1@21a06946]" name="setup" is-config="true" duration-ms="12919" started-at="2019-06-12T18:34:44Z" finished-at="2019-06-12T18:34:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="ornghrmlogin(java.lang.String, java.lang.String)[pri:0, instance:TestScript.TC1@21a06946]" name="ornghrmlogin" duration-ms="0" started-at="2019-06-12T18:34:57Z" data-provider="ornghrmlogin" finished-at="2019-06-12T18:34:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NoClassDefFoundError: org/apache/poi/xssf/usermodel/XSSFWorkbook]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NoClassDefFoundError: org/apache/poi/xssf/usermodel/XSSFWorkbook
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
Caused by: java.lang.NoClassDefFoundError: org/apache/poi/xssf/usermodel/XSSFWorkbook
	at ExcelMethod.XcelFunc.<init>(XcelFunc.java:20)
	at TestScript.TC1.passdata(TC1.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 19 more
Caused by: java.lang.ClassNotFoundException: org.apache.poi.xssf.usermodel.XSSFWorkbook
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 27 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ornghrmlogin -->
        <test-method status="FAIL" signature="teardown()[pri:0, instance:TestScript.TC1@21a06946]" name="teardown" is-config="true" duration-ms="0" started-at="2019-06-12T18:34:57Z" finished-at="2019-06-12T18:34:57Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Process refused to die after 10 seconds, and couldn't taskkill it: Unable to find executable for: taskkill]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Process refused to die after 10 seconds, and couldn't taskkill it: Unable to find executable for: taskkill
	at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:142)
	at org.openqa.selenium.os.ProcessUtils.killProcess(ProcessUtils.java:81)
	at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.destroyHarder(UnixProcess.java:247)
	at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.access$2(UnixProcess.java:246)
	at org.openqa.selenium.os.UnixProcess.destroy(UnixProcess.java:125)
	at org.openqa.selenium.os.CommandLine.destroy(CommandLine.java:155)
	at org.openqa.selenium.firefox.FirefoxBinary.quit(FirefoxBinary.java:246)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.quit(NewProfileExtensionConnection.java:194)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.quit(FirefoxDriver.java:364)
	at org.openqa.selenium.firefox.FirefoxDriver.stopClient(FirefoxDriver.java:310)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:529)
	at TestScript.TC1.teardown(TC1.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.TestRunner.afterRun(TestRunner.java:1023)
	at org.testng.TestRunner.run(TestRunner.java:628)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
Caused by: java.lang.NullPointerException: Unable to find executable for: taskkill
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:251)
	at org.openqa.selenium.os.UnixProcess.<init>(UnixProcess.java:62)
	at org.openqa.selenium.os.CommandLine.<init>(CommandLine.java:38)
	at org.openqa.selenium.os.WindowsUtils.killPID(WindowsUtils.java:178)
	at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:138)
	... 32 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- TestScript.TC1 -->
    </test> <!-- Real Testing -->
  </suite> <!-- Data driven Framework -->
</testng-results>
